CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
Since this is a bugfix, could you change this to say version `2.0.2` and also update the version in the `nextflow.config` file?

Reply:
Updated in ccc46228ef84c99c910bbb520ed25135aec410a4